src/main/java/calculator/config/SeparatorConstants.java

```diff
@@ -0,0 +1,16 @@
+package calculator.config;
+
+import calculator.model.Separator;
+import java.util.List;
+
+public class SeparatorConstants {
+    public static final Separator DEFAULT_SEPARATOR_1 = new Separator(',');
```

Comment:
좀더 명시적인 변수명을 사용했으면 더 좋았을 거라 생각합니다!
```suggestion
    public static final Separator DEFAULT_SEPARATOR_COMMA = new Separator(',');
```